pkg/gardenlet/apis/config/types.go

```diff
@@ -326,6 +326,9 @@ type FluentBit struct {
 
 // Loki contains configuration for the Loki.
 type Loki struct {
+	// Disable the shoot and seed Loki to only use FluentBit with a custom output.
+	// Should be used with fluentBit.output to specify own output.
+	Disable *bool
```

Comment:
You can avoid one negation by using `enable` instead of `disable`. 
```suggestion
	Enabled *bool
```

Reply:
I used the `disable` because the default behavior should be that Loki is enabled. We can use the pointer for check if value is not set to let Loki enabled. Do you prefer this solution?

Reply:
From API point of view `enabled` sounds better. You can still implement defaulting mechanism that keeps loki enabled even when this field is not specified.